### PR TITLE
Feature/112202371 aq

### PR DESCRIPTION
**Pull request by andrewquan**
Had trouble setting up the CSS to make it look prettier. The logic seems to be all there though. I'm new to `accepts_nested_attributes_for` so I had to tinker around with it a bit. Also, I had trouble cleaning out the database for Rspec. Any help on configuring Database Cleaner so we can run proper tests?

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -14,4 +14,4 @@
  *= require_self
  */
 
- @import url(https://fonts.googleapis.com/css?family=Raleway:400,300,600);
+ @import url(https://fonts.googleapis.com/css?family=Raleway:400,300,600);
```
Comment (charliemcelfresh) [inline app/assets/stylesheets/application.css]: let's revert this -- we always want a trailing newline character at the end of every file

File: app/assets/stylesheets/custom.css [hunk 2]
```diff
@@ -18,4 +18,4 @@
 
 .u-full-width {
   width: 100%;
-  box-sizing: border-box; }
+  box-sizing: border-box; }
```
Comment (charliemcelfresh) [inline app/assets/stylesheets/custom.css]: same thing here - let's revert this change

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -48,6 +48,10 @@
     DatabaseCleaner.strategy = :transaction
   end
 
+  config.before(:each, :js => true) do
```
Comment (charliemcelfresh) [inline spec/rails_helper.rb]: we don't use js in this project (yet!), so we don't need this js: true here

File: spec/fabricators/haiku_fabricator.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Fabricator(:haiku) do
```
Comment (charliemcelfresh) [inline spec/fabricators/haiku_fabricator.rb]: We are already using FactoryGirl -- you can get hold of lines_attributes inside of the haiku factory (spec/factories/haikus.rb)

**Comment by charliemcelfresh** [comment]
@andrewquan Not sure why you are seeing these problems with your DB not getting cleaned out. Why don't you take a crack at writing your tests as request specs instead of as controller tests (see spec/requests), leaving the config the way it already is, and see if that fixes your issues?

File: app/views/haikus/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<h1>Haikus!</h1>
+
+<div class="column">
+  <h3>All Haikus</h3>
+  <ul>
+    <% @haikus.each do |haiku| %>
+      <% haiku.lines.each_with_index do |line, index| %>
+        <h5>Title: <%= line.content %></h5>
+        <li><%= index + 1 %>. <%= line.content %></li>
```
Comment (charliemcelfresh) [inline app/views/haikus/index.html.erb]: Let not number the haiku lines

Comment (charliemcelfresh) [inline app/views/haikus/index.html.erb]: Looks like you are repeating your lines twice for each Haiku -- let's just have one  haiku, then two newlines, then another haiku

**Comment by kkchu791** [comment]
@andrewquan i don't know if  this will help but one time it helped me clean my test database by running:

`rake db:test:prepare`
http://stackoverflow.com/questions/15169894/what-does-rake-dbtestprepare-actually-do
